.github/workflows/build.yml

```diff
@@ -115,6 +115,7 @@ jobs:
           mkdir -p "$(dirname "$PREVIEW_ENV_DEV_SA_KEY_PATH")"
           echo "${PREVIEW_ENV_DEV_SA_KEY}" > "${PREVIEW_ENV_DEV_SA_KEY_PATH}"
           gcloud auth activate-service-account --key-file "${PREVIEW_ENV_DEV_SA_KEY_PATH}"
+          gcloud auth configure-docker --quiet
```

Comment:
why is this needed?

Reply:
This is in the parent PR, and just shows here before the rebase. Will be removed once I rebase again